### PR TITLE
Updated FAQ sections

### PR DESCRIPTION
**Pull request by AnaisUrlichs**
Updated the Developer, Validator and Community FAQs.
Already looked over by Stefano, Alex and Mike. Feel free to check again.

### CODE REVIEW

File: docs/community/faq.md [hunk 1]
```diff
@@ -4,9 +4,111 @@ title: Community FAQ
...
+Links with educational content are allowed to be shared in the channels. If you are uncertain, please message an admin or somebody from the core team first; either on [Telegram](https://t.me/cryptonear) or [Discord](http://near.chat/). 
+
+### Where can I provide input?
+Most of our discussions and change or improvement proposals are shared and discussed on our Forum https://commonwealth.im/near/.
+
+# Token Economics
+
+### What are the basic denominations of NEAR?
+We use “NEAR” as a basic denomination. The plural is “NEAR”.
+1 NEAR has 1E18 atto-NEAR. Or 1000 milliNEAR, 1M microNEAR.
```
Comment (bowenwang1996) [inline docs/community/faq.md]: 1 NEAR has 1e24 yotco-NEAR

File: docs/community/faq.md [hunk 2]
```diff
@@ -4,9 +4,111 @@ title: Community FAQ
...
+### When is the MOON?
+Check the lunar calendar yourself. (https://www.timeanddate.com/moon/phases/) 
+
+### Where can I learn more about your token economics?
+Please head over to our economics paper https://nearprotocol.com/papers/economics-in-sharded-blockchain/ 
+
+# Product 
+
+### What products are currently available on NEAR?
+Products include the NEAR-wallet, NEAR-shell, block explorer, IDE, and the actual blockchain (available on TestNet). To learn more, please head over to our documentation.
```
Comment (bowenwang1996) [inline docs/community/faq.md]: Near studio will soon be deprecated. So not sure whether we want to put IDE here

File: docs/roles/developer/faq.md [hunk 3]
```diff
@@ -37,24 +32,33 @@ We don't need GPU support as we are a POS chain and we require very little compu
...
 
-### When renting boxes from the major providers you can Dockerize an application and run it wherever you like without worrying about whether the machine has what you need (as long as the hardware is suitable). With this approach there's very little in terms of requirements for a node to be a validator so how are required libraries / other environmental dependencies for the dApp accounted for?
+### Can I use external libraries in my contract?
 
 Most libraries should still be usable.  However, we do have a size limit for compiled binary of a contract so it is possible that certain large libraries will not be compatible. 
 
 On the other hand, things like interaction with storage is done through our runtime API so it reduces a lot of effort on the backend side of things.
 
+### How do you update a property of an object within a PersistentVector?
```
Comment (bowenwang1996) [inline docs/roles/developer/faq.md]: This and next question seems to be assemblyscript specific. Might be worth noting it.

File: docs/validator/validator-faq.md [hunk 4]
```diff
@@ -23,62 +19,70 @@ More specific steps:
...
-After the transaction is sent, a node that has a private key associated with the public key in the staking transaction must wait until two epochs to become validator. **important**: a node can become a validator only if the amount in the staking transaction is above the seat price defined by the protocol.
+We call staking a process of sending `StakeTransaction` that informs the network that a given account wants to become a validator in upcoming epochs. This particular type of transaction must provide a public key and staking amount. After the transaction is sent, a node that has a private key associated with the public key in the staking transaction must wait until two epochs to become a validator. 
+**important**: a node can become a validator only if the amount in the staking transaction is above the seat price defined by the protocol.
 
-## What is a minimum amount to stake as a validator?
+### What is an Epoch?
+
+An epoch is the interval of time that consists of a fixed number of consensus rounds. Currently, one epoch lasts for about the duration of half a day and is used to
```
Comment (bowenwang1996) [inline docs/validator/validator-faq.md]: there is no guarantee of "fixed number of consensus rounds"

File: docs/validator/validator-faq.md [hunk 5]
```diff
@@ -23,62 +19,70 @@ More specific steps:
...
-## What is an Epoch?
+### What are the responsibilities of a validator?
 
-Epochs represent the interval of time (on the MainNet 12 hours) used by the protocol to:
-- Measure the performance and uptime of validators
-- Collect the bids from new validators
+High level, validators must run node and be mostly online. However, it's very important to keep private keys safe, otherwise adversaries might use them to sign malicious blocks, and trigger the protocol slashing.
 
-## What are the responsibilities of a validator?
+### Is NEAR enforcing liveness fault slashing?
```
Comment (bowenwang1996) [inline docs/validator/validator-faq.md]: Duplicate question here.

File: docs/validator/validator-faq.md (hunk 4 above)
Comment (AnaisUrlichs) [reply docs/validator/validator-faq.md]: That's about what Alex said i.e. that an epoch is not a time duration but a number of consensus rounds. Let me elaborate that there is no guarantee

File: docs/validator/validator-faq.md (hunk 5 above)
Comment (AnaisUrlichs) [reply docs/validator/validator-faq.md]: Removed

File: docs/roles/developer/faq.md (hunk 3 above)
Comment (AnaisUrlichs) [reply docs/roles/developer/faq.md]: Noted, thanks

File: docs/community/faq.md (hunk 2 above)
Comment (AnaisUrlichs) [reply docs/community/faq.md]: Taken out

File: docs/community/faq.md (hunk 1 above)
Comment (AnaisUrlichs) [reply docs/community/faq.md]: Updated

**Pull request merged**